doc/schemas/fundpsbt.request.json

```diff
@@ -32,6 +32,14 @@
     },
     "excess_as_change": {
       "type": "boolean"
+    },
+    "nonwrapped": {
+      "added": "v23.02",
+      "type": "boolean"
+    },
+    "keep_emergency_funds": {
+      "added": "v23.08",
+      "type": "boolean"
```

Comment:
Should this really be a boolean? This kind of assumes that all funds kept back for emergencies are always the same, while think any other HTLC construction (or PTLC) will have a different weight, and different timing requirements. Also in future we might want to keep some funds aside for each channel, or just a single emergency amount that is threaded through multiple channels, and a bool cannot express that. An `amount_msat` to hold back overall is likely more flexible.

Also naming, it's hard :-)

Reply:
Yeah, but I think the internal logic really needs to handle those cases, though. This is simply a hook for "fundchannel" to say "I'm opening an anchor channel this this, so you need the reserve even if you don't think you do".

If we need a more sophisticated emergency reserve, that really needs to be handled elsewhere...

Reply:
OK, I've renamed this arg to "creating_anchor_channel", which is *explicitly* what it means, at least for now.